tests/organization.spec.js

```diff
@@ -61,17 +62,54 @@ describe('organization resolvers', () => {
       URL,
       {
         query: `
-            mutation {
-                updateOrganization(id: "${createdOrgId}", data: {
-                    description: "new description",
-                    isPublic: false
-                    }) {
-                        _id
-                        description
-                        isPublic
-                    }
+          mutation {
+            updateOrganization(
+              id: "${createdOrgId}"
+              data: {
+                name: "test2 org"
+                description: "new description"
+                isPublic: false
```

Comment:
For the variable a random selection should be made for true or false

Reply:
Done.